### PR TITLE
CB-426 Updates for existing Blueprint tests are executing as Cluster Definition

### PR DESCRIPTION
**Pull request by aszegedi**
These are additional changes for CB-388 along with its sub-task:
> CB-426 Double check that all existing Blueprint tests are executing on the renamed cluster definition

Please check these and merge if these are Ok for you!

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/CloudbreakTestSuiteInitializer.java [hunk 1]
```diff
@@ -47,8 +45,8 @@
     @Value("${integrationtest.testsuite.cleanUpOnFailure}")
     private boolean cleanUpOnFailure;
 
-    @Value("${integrationtest.defaultBlueprintName}")
-    private String defaultBlueprintName;
+    @Value("${integrationtest.defaultClusterDefinitionName}")
```
Comment (attilapalfi92) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/CloudbreakTestSuiteInitializer.java]: Are these tests triggered now by jenkins? If yes, are the jenkins jobs updated?

Comment (aszegedi) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/CloudbreakTestSuiteInitializer.java]: I did not find `defaultBlueprintName` reference at hortonworks/cloudbreak-ansible-playbooks repository. So I suppose we do not need additional update.

Comment (attilapalfi92) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/CloudbreakTestSuiteInitializer.java]: alrighty

**Comment by aszegedi** [comment]
There is a blocking issue [CB-585 testInstanceTerminationReplicationError](https://jira.cloudera.com/browse/CB-585), that has came along with [CB-465 Kubernetes v4 related integration test](https://jira.cloudera.com/browse/CB-465) task. So I reported this to @doktoric to further review and investigation.

**Comment by doktoric** [comment]
@aszegedi last circle builds are green on master

**Comment by aszegedi** [comment]
Thx @doktoric! We were able to find the root cause of the mentioned issue. Finally I've closed this, along with the PR.

**Pull request merged**